CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
-## MASTER
```

Comment:
We want to keep this no?

Reply:
I don't think so. We've never really recorded `master`(`main`) branch changelogs. We could start that practice in a new PR IMO